Gemfile

```diff
@@ -108,11 +108,12 @@ gem("byebug", group: [:development, :test])
 # Calling `console` creates irb session in the browser (instead of the terminal)
 gem("web-console", group: :development)
 
+# Use built-in Ruby coverage to generate html coverage file
+gem("simplecov", require: false)
+# generate lcov file to send to Coveralls by Github Actions
+gem("simplecov-lcov", require: false)
 # Automatically track code test coverage
-# Use coveralls_reborn gem instead of coveralls gem
-# With `coveralls` Travis CI runnning with Ubuntu focal gets an SSLError
-# when Travis submits the coverage report to Coveralls
-gem("coveralls_reborn", "~> 0.20.0", require: false)
+gem("coveralls_reborn", "~> 0.23.0", require: false)
```

Comment:
- I don't completely understand how these 3 gems interact. 
- I don't know why I have to explicitly include `simplecov`, but doing so fixed some strange errors. 
See #fd2e4385a4c4a21dae11a04882db41e27698a309